### PR TITLE
My Home: Update copy in the growth summit card

### LINKED ISSUE #54850: My Home: Add Growth Summit card to primary cta area
### What

The growth summit is happening in august 2021 and we'll need to reinstate the promo card for it

### Why

We need to promote the event in my home to current customers.

This card was displayed in 2020 so it should still exist in the code! cc @autumn

### How

Screenshot 2020-07-21 at 14 34 57

Details from the marketing team:

**Title:** The WordPress.com Growth Summit
**Copy:** Join us August 17th (Americas + EMEA) & August 18th (APAC) to get expert advice on how to grow your audience and earn money.
**Button copy:** Register today for $25 USD!
**Link:** https://wordpress.com/growth-summit/?utm_source=my_home&utm_medium=organic_social_media&utm_campaign=myhome&flags=a8c-analytics.on,ad-tracking,google-analytics&ga_optimize=on

Please ping Selena Jackson for final approval.

### PR DESCRIPTION
**Pull request by arthur791004**
#### Changes proposed in this Pull Request

* Update the copy in the growth summit card
image

#### Testing instructions

* Apply D64596-code for your sandbox
* Go to My Home
* Check you can see the card and the copy is correct

Close https://github.com/Automattic/wp-calypso/issues/54850

#### TODO

- [x] Confirm the pager would be the better choice or not
- [x] Confirm should show to non-English user or not
- [x] Confirm when is the last time user can see the card

### CODE REVIEW

File: client/my-sites/customer-home/cards/tasks/growth-summit/index.jsx [hunk 1]
```diff
@@ -11,10 +11,10 @@ const GrowthSummit = () => {
 		<Task
 			title={ translate( 'The WordPress.com Growth Summit' ) }
 			description={ translate(
-				'Learn how to build and grow your site, from start to scale. Come join us for this exclusive two-day virtual event, August 11-13.'
+				'Join us August 17th (Americas + EMEA) & August 18th (APAC) to get expert advice on how to grow your audience and earn money.'
 			) }
-			actionText={ translate( 'Register today for 20% off!' ) }
-			actionUrl="http://www.wordpress.com/growth-summit/"
+			actionText={ translate( 'Register today for $25 USD!' ) }
+			actionUrl="https://wordpress.com/growth-summit/?utm_source=my_home&utm_medium=organic_social_m[…]s=a8c-analytics.on,ad-tracking,google-analytics&ga_optimize=on"
```
Comment (Aurorum) [inline client/my-sites/customer-home/cards/tasks/growth-summit/index.jsx]: Is this link unintentionally truncated? Namely at `social_m[…]s=a8c`.

Comment (arthur791004) [reply client/my-sites/customer-home/cards/tasks/growth-summit/index.jsx]: Oops, good catch! Thank you ❤️

**Review by p-jackson — COMMENTED** [review]
I've tested the growth card and the logic is working as expected (only appears for English speakers, dismissing logic works).

The new illustration feels a bit too big to me, I think the previous one looked a bit better in that regard. I suspect the old SVG used to have some extra margin built it. It's up to the team working on it, so if Selena's happy with it then it's fine with me. I'm just wondering if we should add some extra margin to make it more like the previous illustration.

**Comment by arthur791004** [comment]
Actually, I have already cropped Selena's illustration to make it more like the previous illustration 😂

**Comment by p-jackson** [comment]
> Actually, I have already cropped Selena's illustration to make it more like the previous illustration

😄 No problem, we'll just wait to see what she says

**Comment by jeffgolenski** [comment]
Thanks for jumping into this so quickly folks!

**Review by p-jackson — APPROVED** [review]
:shipit: 

We'll need to deploy this _before_ the backend.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/6294056

Thank you @arthur791004 for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/6294056).

**Pull request merged**